### PR TITLE
Tech/css refactoring

### PR DESCRIPTION
**Pull request by JaiJoPa**
# Refactored CSS for Toolbar

## Description

In this PR, the styles of the toolbar is refactored. We had some float: left and !important keywords in our stylesheets. So I replaced the floats with flexbox, to have some more control over the css and removed the some !important keywords. Some minor refactoring in terms of div-tags were done too. 

## Definition of Done

A task/pull request will not be considered to be complete until all these items can be checked off.

- [x] **All** requirements mentioned in the issue are implemented
- [x] Does match the Code of Conduct and the Contribution file
- [ ] Task has its own **GitHub issue** (something it is solving)
  - [ ] Issue number is **included in the commit messages** for traceability
- [x] **Update the README.md** with any changes/additions made
- [x] **Update the CHANGELOG.md** with any changes/additions made
- [x] **Enough test coverage to ensure that uncovered changes do not break functionality**
- [x] **All tests pass**
- [x] **Descriptive pull request text**, answering:
  - What problem/issue are you fixing?
  - What does this PR implement and how?
- [x] **Assign your PR to someone** for a code review
  - This person _will be contacted **first**_ if a bug is introduced into `master`
- [ ] **Manual testing** did not fail

### CODE REVIEW

**Comment by alexhunziker** [comment]
grafik

So I see the CSS refactoring is here :)

I noticed a small design error: in both FF and Chrome there seem to be too large spaces between the tripple button for the mode changing.

**Comment by alexhunziker** [comment]
grafik
Unfortunately this one is still not alligned on my machine...

**Comment by JaiJoPa** [comment]
> If you want you can also add the two greyed out checkboxes in the download dialog, otherwise i approve the PR.

download_button

you mean checkboxes like this ?

File: visualization/app/codeCharta/ui/filePanel/filePanel.component.html [hunk 1]
```diff
@@ -1,25 +1,25 @@
 <section layout="row" layout-align="center center" class="mode-chooser">
 	<md-button
-		class="state-selector-button left"
+		class="state-selector-button toolbar-selector left"
```
[the remark this answers is not in the document]
Comment (JaiJoPa) [reply visualization/app/codeCharta/ui/filePanel/filePanel.component.html]: You are right. I changed it to mode-chooser

**Comment by alexhunziker** [comment]
grafik

An update on the looks of the scenario button on FF

**Comment by alexhunziker** [comment]
grafik

sorry, there seems to be some new issue :/

**Comment by alexhunziker** [comment]
Ehm... on my machine the loading gif does not appear anymore... is this intentional?

Also i checked the Folder symbol for opening new files: yes it is placed correctly in the center, however it feels a bit too far to the left (but we can leave that too)

**Comment by JaiJoPa** [comment]
> Ehm... on my machine the loading gif does not appear anymore... is this intentional?
> 
> Also i checked the Folder symbol for opening new files: yes it is placed correctly in the center, however it feels a bit too far to the left (but we can leave that too)

Thats weird, it does actually appear on mine, but just for a really short time. It's really fast. Maybe you can see it when you load a bigger map. 
For the Icon-Problem, you are right, it's  a little bit too far to the left. I've adjusted that a bit.

**Comment by alexhunziker** [comment]
> Thats weird, it does actually appear on mine, but just for a really short time. It's really fast. Maybe you can see it when you load a bigger map.

It IS an issue, try it with the Amazon map and exclude something or switch to Delta mode. There on my Firefox the action takes several seconds but no loading gif appears. However it is not an issue with this Branch, but the same behaviour is visible on master. I will open a separate issue for this then.

**Pull request merged**